src/test/java/com/wikia/webdriver/testcases/templateclassification/TemplateClassificationTest.java

```diff
@@ -1,44 +1,60 @@
 package com.wikia.webdriver.testcases.templateclassification;
 
+import com.wikia.webdriver.common.contentpatterns.TemplateTypes;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.elements.oasis.components.templateclassificiation.TemplateClassification;
 import org.testng.annotations.Test;
 import com.wikia.webdriver.common.core.annotations.Execute;
 import com.wikia.webdriver.common.core.annotations.InBrowser;
 import com.wikia.webdriver.common.core.drivers.Browser;
 import com.wikia.webdriver.common.core.helpers.User;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
-import com.wikia.webdriver.elements.oasis.pages.TemplateEditPage;
 import com.wikia.webdriver.elements.oasis.pages.TemplatePage;
 
 @Execute(asUser = User.STAFF, onWikia = "aga")
 @InBrowser(browser = Browser.CHROME)
 @Test(groups = {"templateClassification"})
 public class TemplateClassificationTest extends NewTestTemplate {
 
-  @Test(groups = "templateClassification_openAndClose")
-  public void templateClassification_openAndClose() {
-    new TemplatePage()
-        .open("T")
-        .getTemplateClassification()
-        .open()
-        .close();
-  }
+  @Test(groups = "templateClassification_createTemplateAndChangeItsType")
+  public void templateClassification_createTemplateAndChangeItsType() {
+    TemplatePage templatePage = new TemplatePage();
 
-  @Test(groups = "templateClassification_changeTemplateType")
-  public void templateClassification_changeTemplateType() {
-    new TemplatePage()
-        .open("T")
-        .getTemplateClassification()
-        .open()
-        .changeTemplateType()
-        .close();
-  }
+    TemplateClassification templateClassification = templatePage
+            .createTemplate(TemplateTypes.DEFAULT_TEMPLATE_NAME)
+            .open(TemplateTypes.DEFAULT_TEMPLATE_NAME)
+            .getTemplateClassification()
+            .open()
+            .resetTemplateType()
+            .save();
+
+    String templateType = templateClassification.getTemplateType();
+
+    Assertion.assertTrue(templateType.equals(TemplateTypes.TEMPLATE_TYPE_UNKNOWN), "Template type was reset");
+    PageObjectLogging.logInfo("Template type was reset");
+
+    templateClassification
+            .open()
+            .changeTemplateType(TemplateTypes.TEMPLATE_TYPE_INFOBOX)
+            .save();
+
+    templateType = templateClassification.getTemplateType();
+
+    Assertion.assertTrue(templateType.equals(TemplateTypes.TEMPLATE_TYPE_INFOBOX), "Template type set to Infobox");
+    PageObjectLogging.logInfo("Template type set to: '" + templateType + "'");
+
+    templateClassification
+            .open()
+            .changeTemplateType(TemplateTypes.TEMPLATE_TYPE_QUOTE)
+            .save();
+
+    String oldTemplateType = templateType;
```

Comment:
I did this to rename the variable to be clearer during assertion few lines under